src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -52,11 +52,34 @@ file that was distributed with this source code.
             </div>
             {% if results_count > 0 %}
                 <div class="box-body no-padding">
-                    <ul class="nav nav-stacked sonata-search-result-list">
+                    <ul class="nav nav-stacked sonata-search-result-list products-list">
                         {% for result in current_page_results %}
                             {% set link = admin.getSearchResultLink(result) %}
                             {% if link %}
-                                <li><a href="{{ link }}">{{ admin.toString(result) }}</a></li>
+                                <li class="item no-padding">
+                                    <a class="main pull-left" href="{{ link }}">{{ admin.toString(result) }}</a>
+                                    <span class="matches pull-right">
+                                        {% for name, filter in filters %}
+                                            {% set match = name|split('.')|reduce((carry, v) => carry ? attribute(carry, v) : null, result) %}
+
+                                            {% if match and ((filter.options.case_sensitive and filter.value.value in match)
+                                                or filter.value.value|lower in match|lower)
+                                            %}
+                                            {#
+                                                When case sensitivity issues (like the one present when using MySQL database) were resolved, remove
+                                                the previous condition and uncommment the following one.
+                                                See https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1395.
+                                            #}
+                                            {#{% if match and (((filter.options.case_sensitive or filter.options.case_sensitive is null) and filter.value.value in match)
+                                                or filter.options.case_sensitive is same as(false) and filter.value.value|lower in match|lower)
+                                            %}#}
```

Comment:
It's not clear to me when we can update the code.
The PR: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1395 is already merged.
And the Admin bundle should be aware about the issues from persistence bundles.

Reply:
Plus if we rename the option in the persistence bundle, this won't work. 
The AdminBundle shouldn't rely on the persistence bundle code

Reply:
>The AdminBundle shouldn't rely on the persistence bundle code

This option is set in the `SearchHandler`:
https://github.com/sonata-project/SonataAdminBundle/blob/09e8b7d2ca219ee22c40bad609036f23548e87f4/src/Search/SearchHandler.php#L92

>And the Admin bundle should be aware about the issues from persistence bundles.

We already know that isn't properly respected with some DB platforms.
If I use the commented code instead, this template will respect the "case_sensitive" option, which can lead to not matching some results returned by the persistence bundle (since the persistence bundle is not properly respecting the option).
What do you suggest?

Reply:
I left the right behavior (respecting the "case_sensitive" option), adding this comment in the template:
>Some model managers are not respecting the "sonata_admin.global_search.case_sensitive" configuration option.
If you see no matching filters for a search result, it is probably the reason. In that case, you SHOULD consider using `false` as value for this option.

Is it fine for you?

Reply:
Seems ok like this